lib/galaxy/webapps/galaxy/api/item_tags.py

```diff
@@ -3,71 +3,112 @@
+            )
+            def index(
+                self,
+                trans: ProvidesAppContext = DependsOnTrans,
+                id: DecodedDatabaseIdField = Path(..., title="Item ID", description=f"{tagged_item_id}"),
+            ) -> ItemTagsListResponse:
+                return self.manager.index(trans, tagged_item_class, id)
+
+            @router.get(
+                f"/api/{prefix}/{{id}}/tags/{{tag_name}}",
+                tags=[tagged_item_tag],
+                summary=f"Show tag based on {tagged_item_id}",
+            )
+            def show(
+                self,
+                trans: ProvidesAppContext = DependsOnTrans,
+                id: DecodedDatabaseIdField = Path(..., title="Item ID", description=f"{tagged_item_id}"),
+                tag_name: str = Path(..., title="Tag Name"),
+            ) -> ItemTagsResponse:
+                return self.manager.show(trans, tagged_item_class, id, tag_name)
+
+            @router.post(
+                f"/api/{prefix}/{{id}}/tags/{{tag_name}}",
+                tags=[tagged_item_tag],
+                summary=f"Create tag based on {tagged_item_id}",
+            )
+            def create(
+                self,
+                trans: ProvidesAppContext = DependsOnTrans,
+                id: DecodedDatabaseIdField = Path(..., title="Item ID", description=f"{tagged_item_id}"),
+                tag_name: str = Path(..., title="Tag Name"),
+                payload: ItemTagsCreatePayload = Body(...),
+            ) -> ItemTagsResponse:
+                return self.manager.create(trans, tagged_item_class, id, tag_name, payload)
+
+            @router.put(
+                f"/api/{prefix}/{{id}}/tags/{{tag_name}}",
+                tags=[tagged_item_tag],
+                summary=f"Update tag based on {tagged_item_id}",
+            )
+            def update(
+                self,
+                trans: ProvidesAppContext = DependsOnTrans,
+                id: DecodedDatabaseIdField = Path(..., title="Item ID", description=f"{tagged_item_id}"),
+                tag_name: str = Path(..., title="Tag Name"),
+                payload: ItemTagsCreatePayload = Body(...),
+            ) -> ItemTagsResponse:
+                return self.manager.create(trans, tagged_item_class, id, tag_name, payload)
+
+            @router.delete(
+                f"/api/{prefix}/{{id}}/tags/{{tag_name}}",
+                tags=[tagged_item_tag],
+                summary=f"Delete tag based on {tagged_item_id}",
+            )
+            def delete(
+                self,
+                trans: ProvidesAppContext = DependsOnTrans,
+                id: DecodedDatabaseIdField = Path(..., title="Item ID", description=f"{tagged_item_id}"),
+                tag_name: str = Path(..., title="Tag Name"),
+            ) -> bool:
+                return self.manager.delete(trans, tagged_item_class, id, tag_name)
+
+        return Temp
+
+
+prefixs = {
```

Comment:
```suggestion
prefixes = {
```
;)

This little VSCode extension is a lifesaver, if you don't have it already I totally recommend it https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker